scripts/migrations/README.md

```diff
@@ -41,9 +37,13 @@ pnpm install
 To apply all pending migrations:
 
 ```bash
-pnpm script:db:migrate
+pnpm db:migrate --schema=schema_name
```

Comment:
should we rollback this ?

Reply:
nono, user can still name a different schema if they want (i mean i think flexibility is cool to have) but default is `public` schema